### DISCUSSION TITLE
Way to list all datasources defined in config file

### DISCUSSION THREAD

**Discussion by jeevansai502** [hairyhenderson/gomplate#1286] [Ideas]
Hey there!

We are using gomplate to merge couple of yaml files in an ordered way, All the yaml files are specified as datasources in the config file. Currently we are modifying template file every time a new datasource is added in the config, Is there a way to get all the datasources defined in config so that we can loop through the list and merge.

Created a method to list https://github.com/hairyhenderson/gomplate/pull/1287

Please do check and let me know the feasiblility. 

Thanks.

**Comment by hairyhenderson** [comment]
Sounds reasonable - I've made some comments in the PR.

**Comment by jeevansai502** [comment]
Hey there!

Thank you for the prompt response. Updated the PR, please do let me know if anything else has to be added also Could you please verify if this change works fine before merging.

Thanks again.

**Comment by jeevansai502** [comment]
Hey!

I can see a couple of tasks with message `Expected — Waiting for status to be reported`, how do we trigger these tasks?

**Reply by hairyhenderson** [reply]
My apologies - I've kicked off the build for you...

**Reply by jeevansai502** [reply]
Thank you :)
